rpm/python-cffi.spec

```diff
@@ -0,0 +1,82 @@
+%bcond_with tests
+%bcond_with doc
```

Comment:
I would just drop the doc package, but then again it doesn't do any harm besides the unneeded clutter here